pkg/cmd/extensions/extension.go

```diff
@@ -21,3 +22,20 @@ func (e *Extension) Path() string {
 func (e *Extension) URL() string {
 	return e.url
 }
+
+func (e *Extension) IsLocal() bool {
+	dir := filepath.Dir(e.path)
+	fileInfo, err := os.Lstat(dir)
+	if err != nil {
+		return false
+	}
+	// Check if extension is a symlink
+	if fileInfo.Mode()&os.ModeSymlink != 0 {
+		return true
+	}
+	// Check if extension does not have a git directory
+	if _, err = os.Stat(filepath.Join(dir, ".git")); err != nil {
```

Comment:
I think the check for a missing file should be `errors.Is(err, os.ErrNotExist)`. If it's a different kind of error object, it might be due to causes other than a missing file.